packages/jest-cli/src/__tests__/watch_filename_pattern_mode.test.js

```diff
@@ -142,27 +144,33 @@ describe('Watch mode flows', () => {
     });
   });
 
-  it('Pressing "c" clears the filters', () => {
+  it('Pressing "c" clears the filters', async () => {
     contexts[0].config = {rootDir: ''};
     watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);
 
     stdin.emit(KEYS.P);
+    await nextTick();
 
     ['p', '.', '*', '1', '0']
       .map(toHex)
       .concat(KEYS.ENTER)
       .forEach(key => stdin.emit(key));
 
     stdin.emit(KEYS.T);
-    ['t', 'e', 's', 't']
+    await nextTick();
+
+    [('t', 'e', 's', 't')]
```

Comment:
why the parens? this makes it just return the last one, and we get an array of `['t']` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator)

Reply:
:man_facepalming: :man_facepalming: :man_facepalming: I have no idea how that happen